### PR TITLE
Closing #36: Aboutmsmr

### PR DESCRIPTION
**Pull request by jimmyygit**
image

This commit depends on the development<-maxres PR. idrk how we commit on pending reviews

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,12 +1,71 @@
...
+
+            <h2 className="text-success font-weight-bold"> External Resilience </h2>
+            <p> The ability to define your role within your community. Being active in your community can help create your own identity. </p>
+
+            <Button type="purpleWhite" link="/">
+              <p>LEARN MORE</p>
+            </Button>
+          </div>
+        </div>
+      </section>
```
Comment (functionstackx) [inline pages/index.js]: can you do a component?

**Review by functionstackx — CHANGES_REQUESTED** [review]
make into component

File: pages/index.js (hunk 1 above)
Comment (LemonFace0309) [reply pages/index.js]: true, but it might be better to split it into two components because the two sections differ by quite a bit.

**Review by LemonFace0309 — COMMENTED** [review]
overall really good senpai Jimmy <3 hehe. I haven't been using react bootstrap classes but I'll try to incorporate them in the future

File: pages/index.js [hunk 2]
```diff
@@ -2,12 +2,14 @@ import Head from 'next/head'
...
 
 export default function Home() {
   return (
     <>
       <Landing />
       <Jumbotron />
+      <AboutMSMR />
       <MaxResilience />
 
       <h1> Hello World </h1>
```
Comment (functionstackx) [inline pages/index.js]: can you remove hello world

**Review by functionstackx — CHANGES_REQUESTED** [review]

**Review by functionstackx — APPROVED** [review]

File: pages/index.js (hunk 2 above)
Comment (jimmyygit) [reply pages/index.js]: oh yea

**Pull request merged**